test/utils.py

```diff
@@ -32,6 +33,9 @@ def assert_matching_tables(table1, table2, ignore_headers=False):
         data1 = table1
         data2 = table2
 
+    if isinstance(data1, Table) and isinstance(data2, Table):
+        assert data1.num_rows == data2.num_rows
```

Comment:
I'm realizing this may be causing that error in the Copper tests. I can take a look at those tests to see why this would cause them to fail, since this seems like a valid check.

Reply:
Hey Eliot. Yes, could you take a look? It seems like those tests are failing because the tables have a different number of rows, and this wasn't being checked before.